### ISSUE TITLE
Hydra in Strudel integration ideas

### ISSUE THREAD

**Issue by jarmitage** [tidalcycles/strudel#761]
🎉 Strudel now has a fantastic Hydra integration via https://github.com/tidalcycles/strudel/pull/759

Examples:
- https://strudel.tidalcycles.org/?nbSkcvMg_4fX
- https://strudel.tidalcycles.org/?4Py22Y83TLlC
- https://strudel.tidalcycles.org/?zy8YxWESPl_r

### Integration Ideas
- [ ] What should the Hydra behaviour be when Strudel is stopped? Black screen? Freeze frame? Never stop?
- [ ] Is it possible to feed Strudel audio into Hydra's FFT analyser? (for direct audio viz)
- [ ] It currently doesn't seem possible to use `H()` inside Hydra arrow functions e.g. `.rotate(({time})=>(time*H(p))))`

**Comment by felixroos** [comment]
this would be a first step to allow patterns directly in hydra: https://github.com/hydra-synth/hydra-synth/issues/152

**Comment by felixroos** [comment]
> It currently doesn't seem possible to use H() inside Hydra arrow functions e.g. .rotate(({time})=>(time*H(p))))

that's because H(x) will return a pattern and multiplying patterns with regular values won't work

**Comment by emptyflash** [comment]
> that's because H(x) will return a pattern and multiplying patterns with regular values won't work

I think `.rotate(({time})=>(time*H(p)()))` should work right?

**Comment by felixroos** [comment]
> > that's because H(x) will return a pattern and multiplying patterns with regular values won't work
> 
> I think `.rotate(({time})=>(time*H(p)()))` should work right?

you're right. my last comment was just wrong, because H(x) actually does return a function that returns a value

**Comment by emptyflash** [comment]
>  What should the Hydra behaviour be when Strudel is stopped? Black screen? Freeze frame? Never stop?
Personally in favor of `Ctrl+.` calling Hydra's `hush()`. Because Strudel doesn't have block based evaluation, the only way to stop the music and visuals is to call `hush()` and add `silence` to the end.

**Comment by felixroos** [comment]
> > What should the Hydra behaviour be when Strudel is stopped? Black screen? Freeze frame? Never stop?
> 
> Personally in favor of `Ctrl+.` calling Hydra's `hush()`. Because Strudel doesn't have block based evaluation, the only way to stop the music and visuals is to call `hush()` and add `silence` to the end.

now doing this with https://github.com/tidalcycles/strudel/pull/983
